.github/workflows/jekyll_build_work.yml

```diff
@@ -0,0 +1,28 @@
+name: Jekyll build works
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Jekyll build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '1.8'
+      - uses: kamiazya/setup-graphviz@v1
+      - name: Installs libfontconfig
+        run: sudo apt-get install -y libfontconfig
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
+      - name: Install bundle
+        run: |
+          sudo apt-get -yqq install libpq-dev
+          gem install bundler
+      - name: Bundle install
+        run: |
+          bundle install --jobs 4 --retry 3
+      - name: Run build
+        run: bundle exec jekyll build
```

Comment:
Is it possible to add a step that tests the output from `jekyll build`? I'm thinking of just finding the `index.html` file and doing a string search for whether it contains the SVG diagram from `kramdown-plantuml` and the material icon from `jekyll-material-icon-tag`.

Reply:
Everything is possible!

Reply:
Not finding the output returns in a exit code of 1